src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ContractDetailsPage/index.tsx

```diff
@@ -14,6 +14,13 @@ import { formatDate, Position, useHistory, useCurrentContext } from '@equinor/fu
 import Contract from '../../../../../../models/contract';
 import { getInstances, isInstanceFuture, isInstancePast } from '../../../../orgHelpers';
 import { Link } from 'react-router-dom';
+import ContractAdminTable from '../../components/ContractAdminTable';
+import { useAppContext } from '../../../../../../appContext';
+import useReducerCollection from '../../../../../../hooks/useReducerCollection';
+
+type AccordionOpenDictionary = {
```

Comment:
prefer Record<string, boolean>

Reply:
AccordionOpenDictionary wasn't actually in use, so I removed it.